app/views/categories/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="flex flex-col justify-center  h-full bg-white">
+  <div class="bg-[#3778c2] flex justify-between items-center text-white p-4 py-6">
+      <% if user_signed_in? %>
+       <%= button_to destroy_user_session_path, method: :delete do %>
+          <%= image_tag("https://img.icons8.com/ios/50/import.png", alt: "exit", width: 20, height: 20) %>
+        <% end %>
+      <% end %>
```

Comment:
Well done on the categories page. Categories are created following project requirements and design guidelines!!! 👏🏾 💯 

![image](https://github.com/JakariaJishan/Budget-App/assets/38088818/e7d3d8ce-1195-4bcc-887f-aa58c046f5a7)